<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgepSettingsDialog</class>
 <widget class="QDialog" name="QgepSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="mGbLayout">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="mGbOverrideDefaultProfileTemplate">
        <property name="title">
         <string>Override default profile template</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="mProfileTemplateFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="mPbnChooseProfileTemplateFile">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mGbLayers">
     <property name="title">
      <string>Layers</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="mLblSpecialStructures">
        <property name="text">
         <string>Special structures:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mCbSpecialStructures"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mLblGraphNodes">
        <property name="text">
         <string>Graph nodes:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="mCbGraphNodes"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="mCbGraphEdges"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mLblGraphEdges">
        <property name="text">
         <string>Graph edges:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mCbDevelMode">
     <property name="text">
      <string>Developer mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mBtnBoxOkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mBtnBoxOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>QgepSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mBtnBoxOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>QgepSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
